setup.py

```diff
@@ -21,8 +21,6 @@ def main():
         # not zip-safe.
         zip_safe=False,
         cmdclass=versioneer.get_cmdclass(),
-        tests_require=['six'],
-        test_suite='conda_smithy',
```

Comment:
I personally prefer to just call `pytest` instead of using `setuptools`'s `setup.py test`... In my experience it does not add much to the experience, plus it seems even `setuptools` itself is [moving away](pypa/setuptools#814) from using `setup.py test`.

@jaraco may have something to add on this matter.